rest/data/project.go

```diff
@@ -259,6 +260,27 @@ func (pc *DBProjectConnector) UpdateProjectVars(projectId string, varsModel *res
 	return nil
 }
 
+func (pc *DBProjectConnector) UpdateProjectVarsByValue(toReplace string, replacement string) ([]*db.ChangeInfo, error) {
+	matchingProjects, err := model.GetVarsByValue(toReplace)
+	if matchingProjects == nil {
+		return nil, errors.Wrap(err, "no projects with matching value found")
+	}
+	changes := []*db.ChangeInfo{}
+	for _, project := range matchingProjects {
+		for key, val := range project.Vars {
```

Comment:
Does this work for nested keys, where the value is set of key value pairs and `toReplace` is nested in it?

Reply:
Project varialbes can't be nested; it's a map[string]string, so we don't have to consider this case